### PR TITLE
Budget App

### PR DESCRIPTION
**Pull request by rbd3**
### Change made:
👉 Use ruby gems as software packages system.
👉 Install Ruby on Rails framework.
👉 Use params from browser request in a safe way.
👉 Use preprocessed html file with embedded Ruby code.
👉 Use database migration files to maintain database schema.
👉 Use validations for models.
👉 Set up associations between models.
👉 Build a web app that requires the user to log in.
👉 Use devise gem for authentication.
👉 Add category page
👉 Add transaction page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,167 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo & Video presentation<a name="live-demo"></a> <a name='video'></a>
+<br>
+
+- [Check the live demo, mobile version](https://github.com/rbd3/Budget-App/tree/deploy)
```
Comment (binhussen) [inline README.md]: - kindly, deploy the application using Render, Netlify or GitHub Pages to fit the requirment :+1:

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @rbd3 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Sticked Given Design ✔️
- Well Stractured Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,24 +1,167 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo & Video presentation<a name="live-demo"></a> <a name='video'></a>
+<br>
+
+- [Check the live demo, mobile version](https://budgetuser.onrender.com)
```
Comment (prolajumokeoni) [inline README.md:59]: - [ ] Kindly fix the deployed link, some pages are not working, Kindly check your deployed link works properly, after I input the details to signup I see this error page ⬇️ 
render

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @rbd3 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ Deployed link added

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @rbd3, 
congrats

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**